experiments.tex

```diff
@@ -207,32 +208,35 @@ \subsection{Reducing Communication Cost: Application of Global Arrays}
 The root cause for this behavior was not immediately obvious but by analyzing the flow of data in the context of the Lustre filesystem on which the trajectory files were stored  we could get a sense of possible problem areas.
 
-In order to improve performance we needed to employ strategies to avoid the competition over file access across different ranks when the $\overline{\tcomp}/\overline{\tIO}$ ratio was small.
+In order to improve performance we employed strategies to avoid file access competition across ranks when the $\overline{\tcomp}/\overline{\tIO}$ ratio was small.
 To this end, we experimented with two different ways for reducing the I/O cost:
-splitting the trajectory file into as many segments as the number of processes, thus using file-per-process access, and using the HDF5 file format together with MPI-IO parallel reads instead of the XTC trajectory format.
-We discuss these two approaches and the resulting performance improvements in detail in the following sections.
+\begin{inparaenum}[1)]
+	\item splitting the trajectory file into as many segments as the number of processes, thus using file-per-process access, and
+	\item using the HDF5 file format together with MPI-IO parallel reads instead of the XTC trajectory format.
+\end{inparaenum}
+We discuss these two approaches and their performance improvements in detail in the following sections.
 
 \subsubsection{Splitting the Trajectories (``subfiling'')}
 \label{splitting-traj}
-In all the previous benchmarks all processes were using a shared trajectory file but work on ``subfiling'', a mechanism previously used for splitting a large multi-dimensional global array to a number of smaller subarrays with each one saved in a separate file, reduced the file system control overhead by decreasing the number of processes concurrently accessing a shared file \cite{scalable-IO, scalable-IO1}.
-Because subfiling is known to improve programming flexibility and performance of parallel shared-file I/O we investigated splitting our trajectory file into as many trajectory segments as the number of processes.
-For $N$ processes the trajectory file was split into $N$ segments with each segment having $N_{b}$ frames in it. 
-In this way each  process would access its own trajectory segment file without competition for file accesses. 
+In all the previous benchmarks all processes were using a shared trajectory file. ``Subfiling'' is a mechanism previously used for splitting a large multi-dimensional global array to a number of smaller subarrays. Each smaller array is saved in a separate file. As a result, it reduces the file system control overhead by decreasing the number of processes concurrently accessing a shared file~\cite{scalable-IO, scalable-IO1}.
```

Comment:
I suggest:
`Subfiling is a mechanism previously used for splitting a large multi-dimensional global array to a number of smaller subarrays in which each smaller array is saved in a separate file. Subfiling reduces the file system control overhead by decreasing the number of processes concurrently accessing a shared file~\cite{scalable-IO, scalable-IO1}.`

Reply:
Okay!